### PR TITLE
Sync NetworkCommissioningCluster to Specs

### PR DESCRIPTION
**Pull request by Apollon77**

### CODE REVIEW

**Comment by Apollon77** [comment]
For the sake of iOS testing ... merge if you like and if ok ...

File: src/matter/cluster/NetworkCommissioningCluster.ts [hunk 1]
```diff
@@ -4,25 +4,241 @@
...
+    channel: TlvField(3, TlvUInt16),
+
+    /** Used to differentiate overlapping channel number values across different Wi-Fi frequency bands. */
+    wiFiBand: TlvOptionalField(4, TlvEnum<WiFiBand>()),
+
+    /** Denote the signal strength in dBm of the associated scan result. */
+    rssi: TlvOptionalField(5, TlvInt8),
+});
+
+// TODO: Move to TlvString.ts in matter.js!
```
Comment (mfucci) [inline src/matter/cluster/NetworkCommissioningCluster.ts:112]: Can you give more details about this TODO?

**Review by mfucci — APPROVED** [review]

File: src/matter/cluster/NetworkCommissioningCluster.ts (hunk 1 above)
Comment (Apollon77) [reply src/matter/cluster/NetworkCommissioningCluster.ts:112]: Bildschirm­foto 2023-01-07 um 08 53 44

The relevant field has type "hwaddr" and this is one of these globally defined types in core specs ... (core specs page 391)

Comment (Apollon77) [reply src/matter/cluster/NetworkCommissioningCluster.ts:112]: Thats why I thoight to better have them there? if I'm wrong we can delete the TODO

**Pull request merged**